llvm/lib/Transforms/Instrumentation/MemorySanitizer.cpp

```diff
@@ -4466,6 +4472,20 @@ struct MemorySanitizerVisitor : public InstVisitor<MemorySanitizerVisitor> {
       break;
     }
 
+    // Packed
+    case Intrinsic::x86_avx512_min_ps_512:
+    case Intrinsic::x86_avx512_min_pd_512:
+    case Intrinsic::x86_avx512_max_ps_512:
+    case Intrinsic::x86_avx512_max_pd_512: {
+      // These AVX512 variants contain the rounding mode as a trailing flag.
+      // Earlier variants do not have a trailing flag and are already handled
+      // by maybeHandleSimpleNomemIntrinsic(I, 0) via handleUnknownIntrinsic.
+      bool success = maybeHandleSimpleNomemIntrinsic(I, /* trailingFlags */ 1);
+      (void)success;
```

Comment:
nit: [[maybe_unused]]

Reply:
Thanks, fixed in https://github.com/llvm/llvm-project/pull/124421/commits/56c0267f91967d6db9f9e9f5f93ad63343464230